File: server/scripts/inspect_api.py
import requests

try:
    response = requests.get('http://localhost:3000/api/calculations/eos/aggregated?fiscalYearEnd=2024-12-31')
    data = response.json()
    print("Summary keys:", data['summary'].keys())
    print("Total Leave Deductions:", data['summary'].get('totalLeaveDeductions'))
    print("Total Other Deductions:", data['summary'].get('totalOtherDeductions'))
except Exception as e:
    print(e)

File: server/scripts/inspect_api.js

// Native fetch is available in Node 18+

async function inspect() {
    try {
        const response = await fetch('http://localhost:3000/api/calculations/eos/aggregated?fiscalYearEnd=2024-12-31');
        const data = await response.json();
        console.log("Summary keys:", Object.keys(data.summary));
        console.log("Total Leave Deductions:", data.summary.totalLeaveDeductions);
        console.log("Total Other Deductions:", data.summary.totalOtherDeductions);
    } catch (e) {
        console.error(e);
    }
}

inspect();
